lbz/configuration.py

```diff
@@ -47,6 +64,9 @@ def _parse_value(self, value: Any) -> T:
         except Exception as error:
             raise ConfigValueParsingFailed(self._key, value) from error
 
+    def reset(self) -> None:
```

Comment:
Also, I would write one test for this newly added method - we should avoid adding public methods that are not tested, even if they are simple.